sql/backup.sh

```diff
@@ -184,7 +184,20 @@ pg_rman show detail -B ${BACKUP_PATH} > ${TEST_BASE}/TEST-0011.log 2>&1
 grep OK ${TEST_BASE}/TEST-0011.log | grep FULL | wc -l
 grep ERROR ${TEST_BASE}/TEST-0011.log | grep ARCH | wc -l
 
-echo '###### BACKUP COMMAND TEST-0009 ######'
+echo '###### BACKUP COMMAND TEST-0009 #######'
+echo '###### confirm incremental backup is right #######'
```

Comment:
Is it better to change from "is right" to "working"?

Reply:
Thank you for you comment. I already fix it..

Reply:
OK